deploy/operator-non-root.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: security-operators
```

Comment:
We probably should stick to `seccomp-operator` here…

Reply:
Sure thing - just reverted that across the board.